picard/collection.py

```diff
@@ -36,77 +36,87 @@
 
 
 class Collection(QtCore.QObject):
-    COLLECTION_ADD = 1
-    COLLECTION_REMOVE = 2
 
-    def __init__(self, collection_id, name, size):
+    def __init__(self, collection_id):
```

Comment:
What do you think of passing the MBAPIHelper here instead of accessing the global one on the tagger object? That way we could make the class fully testable (with a mock API helper).

Reply:
Will do